docs/notes/v6.0.0.md

```diff
@@ -0,0 +1,216 @@
+
+### MinIO Job Example (To create a bucket with a Job):
+
+> Assuming that Operator is installed and latest version is used
+
+Deploy `MinIOJob` CRD along with its binded policy:
+
+```yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: mc-job-sa
+  namespace: minio-tenant-1
+---
+apiVersion: sts.min.io/v1alpha1
+kind: PolicyBinding
+metadata:
+  name: mc-job-bingding
+  namespace: minio-tenant-1
+spec:
+  application:
+    serviceaccount: mc-job-sa
+    namespace: minio-tenant-1
+  policies:
+    - consoleAdmin
+---
+apiVersion: job.min.io/v1alpha1
+kind: MinIOJob
+metadata:
+  name: minio-test-job
+  namespace: minio-tenant-1
+spec:
+  serviceAccountName: mc-job-sa
+  tenant:
+    name: myminio
+    namespace: minio-tenant-1
+  commands:
+    - op: make-bucket
+      args:
+        name: memes
+```
+
+3. Job is completed, bucket created:
+
+![bucket creation](images/v6.0.0/image1.png )
+
+### Sidecar container
+
+TODO
+
+### TLS Refactor
+
+TODO
+
+### Operator STS 
+
+API `sts.min.io` is updated from `v1alpha1` to `v1beta1`.
+
+The [PolicyBinding](https://github.com/minio/operator/blob/master/docs/policybinding_crd.adoc#k8s-api-github-com-minio-operator-pkg-apis-sts-min-io-v1alpha1-policybinding)
+CRD will be backward compatible.
+If a resource of type `PolicyBindig` on version `v1alpha` currently exists in the Cluster
+at the moment of the upgrade of Operator to Version 6.0.0, the resource will be automatically updated to version `v1beta1`
+with no manual intervention.
+
+<details>
+  <summary>Example</summary>
+
+Before update:
+```yaml
+apiVersion: sts.min.io/v1alpha
+kind: PolicyBinding
+metadata:
+  name: binding-1
+  namespace: minio-tenant-1
+spec:
+  application:
+    namespace: sts-client
+    serviceaccount: stsclient-sa
+  policies:
+    - test-bucket-rw
+```
+
+After update:
+```yaml
+apiVersion: sts.min.io/v1beta1
+kind: PolicyBinding
+metadata:
+  name: binding-1
+  namespace: minio-tenant-1
+spec:
+  application:
+    namespace: sts-client
+    serviceaccount: stsclient-sa
+  policies:
+    - test-bucket-rw
+```
+</details>
+
+### Fields immutable
+
+`spec.pools.*.volumesPerServer` and `spec.pools.*.servers` fields are immutable once created, this was introduced in 
+https://github.com/minio/operator/pull/2070 on Operator v5.0.15, this is a safety net to prevent potentially dangerous change,
+once a pool is created, and it contains data the pool's number of servers and the number of volumes per server should not be changed.
```

Comment:
```suggestion
`spec.pools.*.volumesPerServer` and `spec.pools.*.servers` fields are immutable once created, ensuring the number of servers or volumes in an existing pool does not change. It was introduced in Operator v5.0.15 with PR https://github.com/minio/operator/pull/2070 and is a safety net to prevent potentially dangerous changes to an existing pool. This also matches similar behavior in Operator Console.
```